python/examples/experimental/NLP_Summarization.ipynb

```diff
@@ -0,0 +1,554 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        ">### 🚩 *Create a free WhyLabs account to get more value out of whylogs!*<br> \n",
+        ">*Did you know you can store, visualize, and monitor whylogs profiles with the [WhyLabs Observability Platform](https://whylabs.ai/whylogs-free-signup?utm_source=whylogs-Github&utm_medium=whylogs-example&utm_campaign=Schema_Configuration)? Sign up for a [free WhyLabs account](https://whylabs.ai/whylogs-free-signup?utm_source=whylogs-Github&utm_medium=whylogs-example&utm_campaign=Schema_Configuration) to leverage the power of whylogs and WhyLabs together!*"
+      ],
+      "metadata": {
+        "id": "5_hazICzT0AX"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Document Summarization Example"
+      ],
+      "metadata": {
+        "id": "phJi2VWRUEio"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/whylabs/whylogs/blob/mainline/python/examples/basic/Schema_Configuration.ipynb)"
+      ],
+      "metadata": {
+        "id": "t_TgL10MUrSZ"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "In this example, we'll look at how we might use whylogs to monitor a document summarization task.\n",
+        "\n",
+        "We'll use [NLTK](https://www.nltk.org) and [BeautifulSoup](https://www.crummy.com/software/BeautifulSoup/) to do some of the basic NLP tasks, so let's install the packages we'll need now."
+      ],
+      "metadata": {
+        "id": "rH4hheWfUt7-"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "yw9ErM4ZTpZo"
+      },
+      "outputs": [],
+      "source": [
+        "%pip install nltk\n",
+        "%pip install bs4\n",
+        "%pip install whylogs-1.1.19-py3-none-any.whl "
```

Comment:
Update when wheel is released

Reply:
you could release a dev version that unblocks this? or put in a version range:
`%pip install "whylogs>1.1.21" --pre`